README.md

```diff
@@ -48,20 +44,36 @@ by running it with the `--help` flag or reading the [rustc dev guide][rustcguide
 [gettingstarted]: https://rustc-dev-guide.rust-lang.org/getting-started.html
 [rustcguidebuild]: https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html
 
-### Building on a Unix-like system
-1. Make sure you have installed the dependencies:
-
-   * `g++` 5.1 or later or `clang++` 3.5 or later
-   * `python` 3 or 2.7
-   * GNU `make` 3.81 or later
-   * `cmake` 3.13.4 or later
-   * `ninja`
-   * `curl`
+### Dependencies
+
+Make sure you have installed the dependencies:
+
+   * `python` 3 or 2.7 (under the name python; python2 or python3 will not work)
    * `git`
-   * `ssl` which comes in `libssl-dev` or `openssl-devel`
+   * OpenSSL (`libssl-dev` or `openssl-devel` on most Unix distros)
+   * XZ (`liblzma-dev` on most distros)
+   * `libffi-dev`
+   * `curl` (not needed on Windows)
    * `pkg-config` if you are compiling on Linux and targeting Linux
+   * `libiconv` (already included with glibc on Debian-based distros)
+
+If building LLVM from source (the default), you'll need additional tools:
+
+* `g++`, `clang++`, or MSVC with versions listed on
+  [LLVM's documentation](https://llvm.org/docs/GettingStarted.html#host-c-toolchain-both-compiler-and-standard-library)
```

Comment:
This changed from the dev-guide; that used a pinned release (LLVM 13.0) but this uses the latest docs. I changed it because I think this README is more likely to get out of date, it doesn't have `<!-- date-check -->` like the dev-guide.